pkg/cilium/cilium.go

```diff
@@ -0,0 +1,34 @@
+package cilium
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// Here we register our own types for Cilium, since we only care about reading one spec field from CiliumNodes.
+// It is easier than trying to use their API library.
```

Comment:
:heart:

Reply:
Credit to Darren for the idea. I wouldn't have thought to do this :)